services/order.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Order, Ticket, User
+from django.db import transaction
+from datetime import datetime
+
+
+def create_order(
+        tickets: list,
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(
+            user=user,
+            created_at=datetime.fromisoformat(date) if date else datetime.now()

```

Comment:
Ensure that the input date string is always in the correct ISO format when using `datetime.fromisoformat(date)` to avoid potential errors.